src/Tribe/Editor/Template/Overwrite.php

```diff
@@ -48,11 +48,15 @@ public function tickets_hook( $ticket_type ) {
 	 */
 	public function include_blocks_in_other_types() {
 		$post_id = get_the_ID();
+		$has_blocks = has_blocks( $post_id );
```

Comment:
Should we check if the function exists first here?

Reply:
We can, it's a function in common and has been there since the initial blocks release

Reply:
It's also the name of a WP function (ours is a duplicate in case the other isn't present from the early days of G)